dev-libs/serdisplib/serdisplib-2.01.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit udev
+
+DESCRIPTION="Library to drive several displays with built-in controllers or display modules"
+HOMEPAGE="http://serdisplib.sourceforge.net"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="deprecated dlo experimental sdl threads tools usb"
+
+# Define the list of valid lcd devices.
+IUSE_LCD_DEVICES=( acoolsdcm ddusbt directgfx displaylink framebuffer glcd2usb
+	goldelox i2c ks0108 l4m lc7981 lh155 nokcol pcd8544
+	remote rs232 sed133x sed153x sed156x ssdoled stv8105 t6963 )
+
+# Add supported drivers from 'IUSE_LCD_DEVICES' to 'IUSE' and 'REQUIRED_USE'
+IUSE="${IUSE} $(printf 'lcd_devices_%s ' ${IUSE_LCD_DEVICES[@]})"
+REQUIRED_USE="${REQUIRED_USE} || ( $(printf 'lcd_devices_%s ' ${IUSE_LCD_DEVICES[@]}) )"
+
+# Specific drivers will need some features to be enabled
+REQUIRED_USE="${REQUIRED_USE}
```

Comment:
Is there any reason to ever enable (deprecated, dlo, experimental, sdl, usb) flags beyond when they are enforced by specific device?

Reply:
Can't say, as I don't know, as I dont use this package. The package itself allows it. Why not? I don't think, this would harm.

Reply:
My point is that it probably be more user-friendly if instead of REQUIRED_USE and additional flags, you'd checked for appropriate LCD_DEVICES and enabled particular features as needed by various drivers.

Reply:
Okay, fine. Let's do it this way. Pushed a new one.
FYI: There is one error left with USE="tools", will look tommorow into it.